cmd/main.go

```diff
@@ -86,21 +85,7 @@ func main() {
 
 	nodeIP := os.Getenv("NODE_IP")
 	nodeName := os.Getenv("NODE_NAME")
```

Comment:
move these 2 lines inside the SanitizeTransportHost() function so that we do not need the extra 2 arguments.
```
transportHost = common.SanitizeTransportHost(transportHost)
```